.eslintrc.json

```diff
@@ -35,6 +35,7 @@
         "import/prefer-default-export": 0,
         "import/no-extraneous-dependencies": 0,
         "react/jsx-fragments": 0,
-        "react/jsx-no-constructed-context-values": 0
+        "react/jsx-no-constructed-context-values": 0,
+        "jsx-a11y/label-has-associated-control":0
```

Comment:
I would imagine you used the input tag without a label tag. To fix issues like this, you could wrap the label tag around the input tag. For example

`<label><input></label>` or 

```
<label htmlFor="name-field">Name </label>
<input id="name-field" ...../> 
```